File: cSharp/easypay/Message.cs
namespace easypay.swisscom.com
{
    /// <summary>
    ///     The response message object represents a server message, which can be extracted and interpreted by the client.
    /// </summary>
    public class MessageItem
    {
        /// <summary>
        ///     Name of the field from the request data model that this message is associated with
        /// </summary>
        public virtual string Field { get; set; }

        /// <summary>
        ///     Symbolic error code identifying the type of error reported by this message
        /// </summary>
        /// <seealso cref="Error" />
        public virtual string Code { get; set; }

        /// <summary>
        ///     The request id, sent within the XRequest-Id http header.
        /// </summary>
        public virtual string RequestId { get; set; }

        /// <summary>
        ///     Localized message describing the nature of the problem reported
        /// </summary>
        public virtual string Message { get; set; }
    }
}

File: Android/EasypayClient/clientlibrary/src/main/java/com/swisscom/easypay/clientlibrary/Message.java
package com.swisscom.easypay.clientlibrary;

/**
 * The response message object represents a server message, which can be extracted and interpreted by the client.
 */
public class Message {

    private String _message;
    private String _field;
    private String _code;
    private String _requestId;

    /**
     * Localized message describing the nature of the problem reported
     */
    public String getMessage() {
        return _message;
    }

    /**
     * Name of the field from the request data model that this message is associated with
     */
    public String getField() {
        return _field;
    }

    /**
     * Symbolic error code identifying the type of error reported by this message
     *
     * @see Error
     */
    public String getCode() {
        return _code;
    }

    /**
     * The request id, sent within the XRequest-Id http header.
     */
    public String getRequestId() {
        return _requestId;
    }

    public void setMessage(String m) {
        _message = m;
    }

    public void setField(String f) {
        _field = f;
    }

    public void setCode(String c) {
        _code = c;
    }

    public void setRequestId(String r) {
        _requestId = r;
    }
}
